docs/articles/storage.md

```diff
@@ -50,6 +50,9 @@ zot can store and serve files from one or more local directories. A minimum of o
 > :pencil2:
 > Remote filesystems that are mounted and accessible locally such as `NFS` or `fuse` are treated as local filesystems.
 
+> :warning:
+> Because Boltdb does not provide concurrent access for writes, multiple instances/replicas of zot are not supported against the same boltdb file when local filesystem is configured.
```

Comment:
This is not the only reason we don't support multiple replicas with local file system. Another reason is there is no locking mechanism zot to block the instances from accessing the same blob file simultaneously. 

Though theoretically you could do it if each zot has exclusive access to a subset of repos from the same storage (not sure how), and you use the redis driver (but we haven't tested this, and haven't included this scenario in the redis docs PR I opened yesterday).

I would move this warning under the BoltDb section, around line 375, and keep only
```
Because BoltDB does not provide concurrent access for writes, multiple instances/replicas of zot are not supported with a BoltDB configuration.
```

Reply:
That works for me - updated.